src/qmlSfmData/SfmDataEntity.cpp

```diff
@@ -240,6 +240,11 @@ void SfmDataEntity::createMaterials()
             } else {
                 gl_PointSize = max(viewportMatrix[1][1] * projectionMatrix[1][1] * pointSize * 0.01 / gl_Position.w, 1.0);
             }
+
+            if (pointSize <= 0.0)
+            {
+                gl_Position = vec4(10.0, 10.0, 10.0, 1.0);
+            }
```

Comment:
I don't understand the point of changing the position when the pointSize is negative or zero.
It needs at least a comment to explain the trick.